### DISCUSSION TITLE
Can't install allin1[train]

### DISCUSSION THREAD

**Discussion by chh-han** [mir-aidj/all-in-one#12] [Q&A]
I tried use `pip install allin1[train]` but failed to found matches

**Comment by trbryant** [comment]
Install the software on my MacBook M2 Pro and it throws a error in line 8 in <module> sys.exit.

**Comment by jordanhenshaw** [comment]
I've been trying to get this thing to work for at least 10 hours now and still have not had success. Tons of issues with numpy no longer accepting np.float. I've just been playing whakamole with every new error that pops up across like 3 different codebases. It's hell. This stuff is far from production ready.

**Reply by jordanhenshaw** [reply]
I finally got mine to work! And it works! 

I had to remove and reinstall numpy to use I think version 1.23 or something. I also had to add/link the C headers because I was working in Blender's python environment. I also had to do a bunch of funny business with all kinds of other dependencies that were not mentioned here, and each of them had their own complaints. I also had to make an edit to one of the madmom scripts I think, something about "from something import something_else", I had to make it be "from something.abc import something_else".

Definitely not something you want to do if you don't know how to use a command line.
